extension/solarwindsextension/README.md

```diff
@@ -0,0 +1,41 @@
+# Solarwinds Extension
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]  |
+| Distributions | [] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/solarwinds/solarwinds-otel-collector?query=is%3Aissue%20is%3Aopen%20label%3Aextension%2Fsolarwinds%20&label=open&color=orange&logo=opentelemetry)](https://github.com/solarwinds/solarwinds-otel-collector/issues?q=is%3Aopen+is%3Aissue+label%3Aextension%2Fsolarwinds) [![Closed issues](https://img.shields.io/github/issues-search/solarwinds/solarwinds-otel-collector?query=is%3Aissue%20is%3Aclosed%20label%3Aextension%2Fsolarwinds%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/solarwinds/solarwinds-otel-collector/issues?q=is%3Aclosed+is%3Aissue+label%3Aextension%2Fsolarwinds) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
+<!-- end autogenerated section -->
+
+SolarWinds Extension is required sidecar for the [Solarwinds Exporter](../../exporter/solarwindsexporter).
+
+It provides these features:
+
+- Hearbeat signal
+  - It's a standard metric: `sw.otecol.uptime`
+    - The value is a time from the start of the collector in seconds
+  - This signal is necessary for SWO to detect the collector when installed and to determine if it's still alive.
+  - It also contains some additional informations as resource attributes for SWO:
+    - Collector name: `sw.collector.name`
+
+## Getting Started
+
+You just need to include the SolarWinds Extension in your extension definitions and provide the following configuration:
+
+```yaml
+extensions:
+  solarwinds:
+    token: "YOUR-INGESTION-TOKEN"
+    data_center: "na-01"
+    collector_name: "Collector Display Name"
+```
+- `token` (mandatory) - You can generate your token in your SWO (SolarWinds Observability) account under _Settings / API Tokens / Create API Token_. The type is "Ingestion". You can find the complete documentation [here](https://documentation.solarwinds.com/en/success_center/observability/content/settings/api-tokens.htm).
+- `data_center` (mandatory) - Data center is the region you picked during the sign-up process. You can easily see in URLs after logging in to SolarWinds Observability SaaS - it's either `na-01`, `na-02` or `eu-01`. Please refer to the [documentation](https://documentation.solarwinds.com/en/success_center/observability/content/system_requirements/endpoints.htm#Find) for details.
+- `collector_name` (mandatory) - this is the name under which the collector will be shown in the SWO UI. Doesn't have to be unique.
```

Comment:
I'd skip the part about collector being shown in SWO. At this point, the name is solely for differentiation or filtering of data